backend/lib/models/IndividualUser.js

```diff
@@ -23,6 +23,10 @@ function updateAuthorLastName(lastName) {
 
 const individualUserSchema = new Schema(
   {
+    authId: {
```

Comment:
@joaofnds I actually have more questions about this one. Is this supposed to be like `auth0|5ecd...` from Auth0? @robinv85 do we need this at all since it's the app metadata mongo id on the Auth0 side that is linking these. Also, I don't see where this would come from in `/users.js` `post("/"...`. We get _id: userId from plugins/auth and everything else from POST body.

Once we clarify this and the questions about email storage I think I'm good to merge.

Reply:
Ah yeah I didn't notice this.
Either we have to auto generate the primary id for organizations
Or we have to create a user account for an organization with an email
but, can it be the same as an individual user? In that case we cannot have 2 user accounts
unless we have a separate dtabase in Auth0 for organizations as well
but we cannot login with email / password and match it to both a individual and an organization user anyway
So we always login as individual, which can be the admin / owner of an organization...
and we don't really have organizations in Auth0, unless in a separate databse? 
Maybe we need to double check to coordinate product requirements with technical possibilities and limitations here.

Reply:
oh maybe i have misunderstood, I though this was for the organization
but if it's only for the individual user then I think it's the same as the mongo_id
this is generated by Auth0 as a `BSON.ObjectId` so we can use it as the `_id` primary key in the users database, we could also store the Auth0 id as well if we need to call the Auth0 API from our backend, so maybe it's this?

Reply:
and this also comes from the JWT in that case which is in `req.user.sub` (subject of the JWT, but containing the Auth0 id which depends on the login method)

Reply:
If anything it would be the `req.user.sub`.

I'm just not sure why we need to store it on the model. But if we do we need to set it somewhere like, for example in users.js app.post:

`const userData = {
...req.body,
_id: req.userId,
authId: req.user.sub
}`

Reply:
yeah I think it's best that we save that also, maybe it's not used at the moment, but what if we need to call the Auth0 API with a user id that logged in, I guess we could use the access token, but usually we use the server token and the access token might need to be changed if it's not for use in the Auth0 API directly, unless we limit scope to read only.